cmd/dfns_example/config.yaml.example

```diff
@@ -0,0 +1,16 @@
+api:
+  appId: "ap-1<redacted>cqgi"
+  appSecret: null
```

Comment:
can we not show this in example? we don't have support for it, leaving it in will just confuse people.

Reply:
SHould I remove appSecret? Is it still used?

Reply:
theoretically used for "server-side Applications" (as Cody called them), but in TS SDK, the param is added but the code actually doesn't use it I believe, so it's not even supported. We advocate for people to use Client-Side Applications even on servers (even if the name doesn't make sense).

SO you can remove if you don't use it